src/fcs.jl

```diff
@@ -55,7 +63,7 @@ end
 Recompute the FCS for a record.
 """
 function compute_fcs(x::PcapRecord)
-    data_no_fcs = UnsafeArray{UInt8, 1}(x.data.pointer, x.data.size .- ETHERNET_FCS_LENGTH)
+    data_no_fcs = UnsafeArray{UInt8,1}(x.data.pointer, x.data.size .- ETHERNET_FCS_LENGTH)
```

Comment:
This looks like it's just wrongly reformatted, are you running auto formatter?
```suggestion
    data_no_fcs = UnsafeArray{UInt8, 1}(x.data.pointer, x.data.size .- ETHERNET_FCS_LENGTH)
```

Reply:
Ugh, yes, apologies. I'll try and get my autoformatter to stop doing this.